sct-commons/src/main/java/org/lfenergy/compas/sct/commons/DoTypeService.java

```diff
@@ -25,4 +32,66 @@ public Optional<TDOType> findDoType(TDataTypeTemplates tDataTypeTemplates, Predi
         return getFilteredDoTypes(tDataTypeTemplates, tdoTypePredicate).findFirst();
     }
 
-}
+    public List<DoLinkedToDa> getAllSDOLinkedToDa(TDataTypeTemplates dtt, TDOType tdoType, DoLinkedToDa filtredDoLinkedToDa) {
```

Comment:
`filtredDoLinkedToDa` is used as a template, not a filter.
I suggest `doLinkedToDaTemplate`